File: Programming Basics/Conditional Statements Advanced - Exercises/07.HotelRoom.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        string month = Console.ReadLine();
        int nights = int.Parse(Console.ReadLine());

        double studioPrice;
        double apartmentPrice;

        switch (month)
        {
            case "May":
            case "October":
                studioPrice = 50;
                apartmentPrice = 65;

                if (nights > 7 && nights < 14)
                    studioPrice -= studioPrice * 0.05;

                else if (nights > 14)
                {
                    studioPrice -= studioPrice * 0.30;
                    apartmentPrice -= apartmentPrice * 0.10;
                }

                Console.WriteLine($"Apartment: {apartmentPrice * nights:f2} lv.");
                Console.WriteLine($"Studio: {studioPrice * nights:f2} lv.");
                break;
            case "June":
            case "September":
                studioPrice = 75.20;
                apartmentPrice = 68.70;

                if (nights > 14)
                {
                    studioPrice -= studioPrice * 0.20;
                    apartmentPrice -= apartmentPrice * 0.10;
                }

                Console.WriteLine($"Apartment: {apartmentPrice * nights:f2} lv.");
                Console.WriteLine($"Studio: {studioPrice * nights:f2} lv.");
                break;
            case "July":
            case "August":
                studioPrice = 76;
                apartmentPrice = 77;

                if (nights > 14)
                    apartmentPrice -= apartmentPrice * 0.10;

                Console.WriteLine($"Apartment: {apartmentPrice * nights:f2} lv.");
                Console.WriteLine($"Studio: {studioPrice * nights:f2} lv.");
                break;
        }
    }
}

File: Programming Basics - Python/Conditional Statements Advanced - Exercises/07.HotelRoom.py
month = input()
duration = int(input())

studio_price = 0
apartment_price = 0

if month in ['May', 'October']:
    studio_price = 50
    apartment_price = 65

    if duration > 14:
        studio_price *= 0.70
        apartment_price *= 0.90

    elif duration > 7:
        studio_price *= 0.95

elif month in ['June', 'September']:
    studio_price = 75.20
    apartment_price = 68.70

    if duration > 14:
        studio_price *= 0.80
        apartment_price *= 0.90

elif month in ['July', 'August']:
    studio_price = 76
    apartment_price = 77

    if duration > 14:
        apartment_price *= 0.90

print(f'Apartment: {duration * apartment_price:.2f} lv.')
print(f'Studio: {duration * studio_price:.2f} lv.')